### ISSUE TITLE
Plot Merging and Claiming Not Working! 

### ISSUE THREAD

**Issue by nyonee** [IntellectualSites/PlotSquared#2064]
Debug paste link: https://pastebin.com/raw/FZujPvaX

Description of the problem:
Every time I download and run PlotSquared I get this bug where the chunks don't fully load or something similar that has to do with claiming plots and merging them, for all that I know I would think it's something that has to do with the chunks loading in the world or something like that since only some parts of the plot are affected when I claim a Plot, ex: I set the plot border/slabs of unclaimed to a normal stone slab, then on claim it should set to a white quartz slab, which only happens on some parts of the plot and doesn't affect the whole plot. It has also happened to me when I claimed a plot instead of the sign being signed and there it was a slab instead? Really weird way this plugin is working with me right now, it also only signed the plot 20% of the time, there was a small chance it would put a slab another chance it puts a stone slab and another a sign and only once or twice out of trying countless times that it actually signed the plot. I tried re installing the plugin from bukkit and now I get the glitch (might have happened because of multiworld) before I didn't do this for the previous glitch but I tried doing /multiworld load Plots (<-- the name of the world the plots are located in) because when I did /multiworld list it showed Plots was unloaded, so then what now happens is that the signs now always work but the slabs do not even change at all and now when I try to merge plots it says "[P2] Plots have been merged!" but really nothing changes at all! the path doesn't do anything and neither does the slabs or anything, in other words before it wanted to load but weirdly and now it really doesn't want to load at all! It's been about a day with this and I looked all over the place and haven't found a fix!

File of worlds in the plugin folder: https://pastebin.com/raw/2Bg0L9EE

How to replicate:
The thing is I wish this was gone but it replicates it self every time I download and run PlotSquared
I will include some screen shots to show you how it looks in game: 
![2018-08-15_11 08 15](https://user-images.githubusercontent.com/42404811/44141355-33af6e60-a07d-11e8-887f-7674c6221190.png) 
![2018-08-15_11 08 52](https://user-images.githubusercontent.com/42404811/44141383-4aafd38e-a07d-11e8-935b-c6a0307dccac.png)
![2018-08-15_11 09 04](https://user-images.githubusercontent.com/42404811/44141404-55da7566-a07d-11e8-82b8-201a75428893.png)
![2018-08-15_11 14 44](https://user-images.githubusercontent.com/42404811/44141435-6b3ad040-a07d-11e8-95fc-a3c85aafc464.png)
![2018-08-15_11 14 55](https://user-images.githubusercontent.com/42404811/44141485-90a5de60-a07d-11e8-9982-56d72e5eaccb.png)
![2018-08-15_11 15 01](https://user-images.githubusercontent.com/42404811/44141503-9c5f3ea4-a07d-11e8-8067-d85e5076194f.png)
![2018-08-15_11 15 40](https://user-images.githubusercontent.com/42404811/44141520-adc8365a-a07d-11e8-99f0-60175f02e4be.png)

Checklist:

- [X] I included a `/plot debugpaste` link:
- [x] I made sure there are no duplicates of this report [(Use Search)](https://github.com/IntellectualSites/PlotSquared/issues?utf8=%E2%9C%93&q=is%3Aissue)
- [X] I made sure I am using an up-to-date version of PlotSquared
- [X] I Made sure the bug/error is not caused by any other plugin <-- Don't know what plugin could cause this and I have over 25 plugins which would take forever.

**Comment by collinator24** [comment]
Check your config files, and change the number of plots allowed by someone/merge plot limit

**Comment by chedadmin** [comment]
Having the exact same problem. Did you manage to fix it?
